NutriScan/NutriScan/Components/Informative Components/Model/IconType.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  IconType.swift
+//  NutriScan
+//
+//  Created by Elena Diniz on 10/14/25.
+//
+import SwiftUI
+
+enum IconType {
+    case asset(name: String)
```

Comment:
Ficou muito bom as refatorações, uma sugestão seria ter um enum para os assets name tbm, como se fosse um "de-para" para evitar de usar a string hardcoded do nome dos icones exemplo:
```swift
enum AssetName {
case brokenHeart = "Broken Heart Icon"
}
```

E quem for usar o componente vai usar apenas o case, evitando de escrever o nome do icone em vários locais do código
```swift
IconCircleView(icon: .asset(name: .brokenHeart), backgroundColor: .iconBackground)
```

Assim, se algum dia precisar mudar o nome do icone, só precisa alterar no valor do enum